index.html

```diff
@@ -4,9 +4,152 @@
+				<li><a href="#" class="nav-item-desktop">Join</a></li>
+				<li><a href="#" class="nav-item-desktop">Sponser</a></li>
+				<li><a href="#" class="nav-item-desktop">News</a></li>
+				<li class="nav-item-desktop nav-btn">
+           <button class="top-btn">JS Hackathon</button>
+				</li>
 
+			</ul>
+		</nav>
+	</header>
+  <div class="container show" id="main-body">
+		<header class="mobile">
+			<nav class="navbar">
+				<div>
+					<img src="assets/images/icons/hamburger-icon.svg" alt="menu icon" class="menu-icon" id="menu-icon"></div>
+			</nav>
+		</header>
+		<section class="hero">
+		  <div class="hero-heading">
+				<span><p class="supporting-text">"Hello coding world"</p></span>
+				<h1 class="primary-text">Javascript Devs Hackathon 2023</h1>
+			</div>
+			<div>
+				<p class="description-hero">
+					Javascript Hackathon is an annual weekend long beginer friendly hackathon series for Javascript enthusiasts with developers from all over the world and different skill levels.
+				</p>
+			</div>
+			<div class="date-venue">
+				<h6 class="date">2023.02.04(SAT) ~ 5(SUN)</h6>
+				<p class="description venue">@ National Museum of Karea, Art Center Nabi and more
+        </p>
+			</div>
+		</section>
+		<section class="program">
+			<h2 class="main-title dark">Main Program</h2>
+			<hl class="divider"></hl>
+			<div class="program-cards-container">
+				<div class="program-card">
+					<img src="assets/images/icons/program-icon.png" alt="Program icon" class="program-icon">
+					<h3 class="program-title">Lecture</h3>
+					<p class="program-description">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+				</div>
+				<div class="program-card">
+					<img src="assets/images/icons/program-icon-2.png" alt="Program icon" class="program-icon">
+					<h3 class="program-title">CC Exhibition</h3>
+					<p class="program-description">Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+				</div>
+				<div class="program-card">
+					<img src="assets/images/icons/program-icon-3.png" alt="Program icon" class="program-icon">
+					<h3 class="program-title">Forum</h3>
+					<p class="program-description">Have the time to share your thoughts and opinions with experts for each topic.</p>
+				</div>
+				<div class="program-card">
+					<img src="assets/images/icons/program-icon-4.png" alt="Program icon" class="program-icon">
+					<h3 class="program-title">Workshop</h3>
+					<p class="program-description">Try creating your own work using open source license rather than just watching at it.</p>
+				</div>
+				<div class="program-card">
+					<img src="assets/images/icons/program-icon-5.png" alt="Program icon" class="program-icon">
+					<h3 class="program-title">CC ignite</h3>
+					<p class="program-description">get opportunities to network with CC affiliates around the world, also after the summit.</p>
+				</div>
+			</div>
```

Comment:
- in your "Main Program" desktop version, kindly ensure that the individual cards have an on-hover that has a highlight effect on the card when hovered over.

![image](https://user-images.githubusercontent.com/60612329/212443462-871db589-855c-406f-a46c-5ed61187ecd2.png)

-  On Mobile view, kindly add some padding/margin-top at the Main program section to give it some spacing which will make it match the provided Behance design.

![image](https://user-images.githubusercontent.com/60612329/212443518-f56a8250-16ca-4912-a3a0-04a13716fdc8.png)

- the mobile version of your project is breaking in both home and about page please fix this and submit again

Reply:
Thank you for your feedback. This issue was also caused because of the background image and style file not loading properly because of the incorrect reference to the relative path of the assets folder, I have now fixed it.